low-level/sspl_ll_d

```diff
@@ -284,30 +283,29 @@ def main(conf_reader, systemd_support):
         _dropPrivileges("sspl-ll")
 
     global sspl_settings_configured_groups
-    sspl_settings_configured_groups |= SSPL_SETTINGS.keys()
+    env_based_sensors = Conf.get(SSPL_CONF,
+        f'{SSPL_SETTING}>{ENV_BASED_SENSORS}', [])
+    sspl_settings_configured_groups |= set(env_based_sensors)
 
     conf_reader = ConfigReader()
-    for group in SSPL_SETTINGS.keys():
+    for group in env_based_sensors:
```

Comment:
why only env based sensors handled here and not all like existing code ? Any sensor with 'monitor' = false can be ignored easily, whatever may be the reason.

Reply:
Yes,  added this change at line 408. checking 'monitor' value for each sensor.